### PR TITLE
Fix/ads user experience

### PR DESCRIPTION
**Pull request by sambua**

### CODE REVIEW

File: src/dynamics/video_player/player/states.js [hunk 1]
```diff
@@ -867,6 +874,12 @@ Scoped.define("module:VideoPlayer.Dynamics.PlayerStates.PlayAd", [
             next: function(state) {
                 this.dyn.set("playing_ad", false);
                 inherited.next.call(this, state);
+            },
+
+            pause: function() {
+                this.dyn.player.pause();
+                // Fid ads player container to the content player container
+                this.dyn.trigger("resize");
```
Comment (ntshcalleia) [inline src/dynamics/video_player/player/states.js:882]: Hi @sambua, we shouldn't manually trigger `resize`, this event should only be triggered by the ResizeObserver.

```suggestion
            }
```

**Comment by sambua** [comment]
Hi @ntshcalleia, I'm not remember now, but resize trigger fixed something. I'll remove it for now, and if find reason why it was here will let you know.

**Comment by ntshcalleia** [comment]
> Hi @ntshcalleia, I'm not remember now, but resize trigger fixed something. I'll remove it for now, and if find reason why it was here will let you know.

The changes for ZIG-3145 probably fixes the issue you were having 🙂

**Comment by ntshcalleia** [comment]
Hi @sambua, can we disable the tooltip for now?
I also noticed that if I mute/unmute the audio during midroll or postroll the changes aren't reflected on the video. Tested on sandbox with this adtagurl: "https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fads%3Fiu%3D%2F21775744923%2Fexternal%2Fvmap_ad_samples%26sz%3D640x480%26cust_params%3Dsample_ar%253Dpremidpost%26ciu_szs%3D300x250%26gdfp_req%3D1%26ad_rule%3D1%26output%3Dvmap%26unviewed_position_start%3D1%26env%3Dvp%26impl%3Ds%26cmsid%3D496%26vid%3Dshort_onecue%26correlator%3D"

**Review by ntshcalleia — CHANGES_REQUESTED** [review]
Hi @sambua, can we disable the tooltip for now?
I also noticed that if I mute/unmute the audio during midroll or postroll the changes aren't reflected on the video. Tested on sandbox with this adtagurl: "https%3A%2F%2Fpubads.g.doubleclick.net%2Fgampad%2Fads%3Fiu%3D%2F21775744923%2Fexternal%2Fvmap_ad_samples%26sz%3D640x480%26cust_params%3Dsample_ar%253Dpremidpost%26ciu_szs%3D300x250%26gdfp_req%3D1%26ad_rule%3D1%26output%3Dvmap%26unviewed_position_start%3D1%26env%3Dvp%26impl%3Ds%26cmsid%3D496%26vid%3Dshort_onecue%26correlator%3D"

**Comment by sambua** [comment]
Hi @ntshcalleia I've made some improvements on first detection and for now commented tooltip related addition. Please review when you have a time

To be able test with vmap you can use /static/demos/embed/test-player.html?blk=1&ad=vmap&ap=1 where ad=vmap

**Review by ntshcalleia — APPROVED** [review]

**Pull request merged**